posthog/session_recordings/queries/session_recording_list_from_filters.py

```diff
@@ -463,15 +429,17 @@ def _event_predicates(self):
     def _select_from_events(self, select_expr: ast.Expr) -> ast.SelectQuery:
         return ast.SelectQuery(
             select=[select_expr],
-            select_from=ast.JoinExpr(table=ast.Field(chain=["events"])),
+            select_from=ast.JoinExpr(
+                table=ast.Field(chain=["events"]),
+            ),
             where=self._where_predicates(),
             having=self._having_predicates(),
             group_by=[ast.Field(chain=["$session_id"])],
         )
 
     def get_query_for_session_id_matching(self) -> ast.SelectQuery | ast.SelectUnionQuery | None:
         use_poe = poe_is_active(self._team) and self.person_properties
-        if self._filter.entities or use_poe:
+        if self._filter.entities or self.event_properties or use_poe:
```

Comment:
this is the thing we need to make sure that we're not joining from replay events to events

TBH we should follow up and remove that auto join it's too dangerous